apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/Util.kt

```diff
@@ -14,6 +14,8 @@ private val JAVA_RESERVED_WORDS = arrayOf(
 
 fun String.escapeJavaReservedWord() = if (JAVA_RESERVED_WORDS.contains(this)) "${this}_" else this
 
+fun String.toJavaBeansSemanticNaming() = "get${this.capitalize()}"
+
```

Comment:
Is it always going to be get? What if we have a variable named enabled, shouldn't it be `isEnabled`?

Reply:
Good point -- Booleans ought to be `isFoo` rather than `getFoo` to be most semantic